factory/__init__.py

```diff
@@ -5,24 +5,30 @@
 from slackclient import SlackClient
 from interface.slack import Bot
 from webhook.webhook import WebhookHandler
+from github import Github, GithubException
 import toml
-
+from interface.github import GithubInterface
 import logging
 
 
-def make_core(config):
+def make_core(config, gh=None):
     """
     Initialize and returns a :class:`command.core.Core` object.
 
     :return: a new ``Core`` object, freshly initialized
     """
+    slack_api_token, github_api_token, github_organization = "", "", ""
     if not config['testing']:
-        slack_api_token = toml.load(config['slack']['creds_path'])['api_token']
-    else:
-        slack_api_token = ""
+        slack_api_token += toml.load(
```

Comment:
Out of curiosity, is there a reason you used `+=`? (I don't think it's a problem, just curious about the reason for the change.)

Reply:
I have no idea why I did, I can remove if needed?